src/main/java/org/apache/sysds/runtime/matrix/data/MatrixBlock.java

```diff
@@ -1332,52 +1352,6 @@ else if( !sparse && denseBlock!=null ) { //DENSE
 		
 		return 0; //empty block
 	}
-	
-	/**
-	 * Basic debugging primitive to check correctness of nnz.
-	 * This method is not intended for production use.
-	 */
-	public void checkNonZeros() {
```

Comment:
where have these debugging primitives (check nnz, check sparse representation) been moved? Please keep them in the code base.